zzzz_modular_occulus/code/modules/reagents/reagents/other.dm

```diff
@@ -5,4 +5,387 @@
 */
 
 /datum/reagent/resuscitator
-		reagent_type = "Stimulator" //For some reason this doesn't inherit a reagent type unless we do this...????
\ No newline at end of file
+		reagent_type = "Stimulator" //For some reason this doesn't inherit a reagent type unless we do this...????
+
+
+/datum/reagent/liquid_dark_matter
+	name = "Liquid Dark Matter"
+	description = "Sucks everything into the detonation point."
+	reagent_state = LIQUID
+	color = "#210021"
+	taste_description = "compressed bitterness"
+
+/datum/reagent/sorium
+	name = "Sorium"
+	description = "Sends everything flying from the detonation point."
+	reagent_state = LIQUID
+	color = "#5A64C8"
+	taste_description = "air and bitterness"
+
+/datum/reagent/tartrate
+	id = "tartrate"
+	name = "Chlorified Tartrate"
+	description = "Mix with enough Aluminum Nitrate for tasty results!"
+	color = "#EA67B1"
+
+datum/reagent/nitrate
+	id = "nitrate"
+	name = "Nitrate"
+	description = "Nitrate, not that interesting."
+	reagent_state = LIQUID
+	color = "#D8DFE3"
+
+/datum/reagent/aluminum_nitrate
+	id = "aluminum_nitrate"
+	name = "Aluminum Nitrate"
+	description = "Aluminum Nitrate, now that's interesting!"
+	reagent_state = LIQUID
+	color = "#E1CFE3"
+
+
+/datum/reagent/babelizine
+	id = "babelizine"
+	name = "Babelizine"
+	description = "Similar to an enzyme produced by the incredibly rare Babel Fish, might have great linguistic applications"
+	color = "#E5F58E"
+	metabolism = 0.2 * REM
+
+/datum/reagent/babelizine/on_mob_life(var/mob/living/M as mob, var/alien)
+	M.universal_understand = 1
+	..()
+
+/datum/reagent/babelizine/on_remove(var/atom/A)
+	if(istype(A, /mob/living))
+		var/mob/living/M = A
+		M.universal_understand = 0
+		to_chat(M, "<span class='warning'>You no longer feel attuned to the spoken word.</span>")
+
+/datum/reagent/babelizine/on_mob_death(var/mob/M)
+	holder.remove_reagent(src.id, src.volume)
+
+
+/datum/reagent/malaxitol
+	id = "malaxitol"
+	name = "Malaxitol"
+	description = "Analysis indicates it could greatly speed up the rate at which other reagents are metabolized"
+	color = "#A155ED"
+	metabolism = 2 * REM
+
+/datum/reagent/malaxitol/on_mob_life(var/mob/living/M as mob, var/alien)
+	for(var/datum/reagent/R in M.reagents.reagent_list)
+		if(R.id == src.id)
+			continue
+		R.on_mob_life(M, alien)
+
+	..()
+
+/datum/reagent/delight			//Joyflower chem, used to make Bliss
+	name = "Delight"
+	id = "delight"
+	description = "A chemical naturally made in Joy Flowers, its known to make people smile"
+	taste_description = "sweet"
+	reagent_state = LIQUID
+	color = "#ffc0cb"
+	metabolism = REM * 0.5
+	overdose = REAGENTS_OVERDOSE * 2
+	sanity_gain = 0.5
+	addiction_chance = 5
+
+/datum/reagent/bliss
```

Comment:
This should be listed as a drug/stim, so insight can be gained off it.

Reply:
Easy fix, can do.